### DISCUSSION TITLE
Tekton Results Logo Poll!

### DISCUSSION THREAD

**Discussion by adambkaplan** [tektoncd/results#451] [Polls]
The ~~CD~~ Linux Foundation artists have provided 5 different concepts for a project logo! These keep the spirit and branding of the other Tekton project logos - see [cdfoundation/artwork/tekton/additional-artwork](https://github.com/cdfoundation/artwork/tree/main/tekton/additional-artwork) for the others.

Below is v1 of their ideas - please take a look and vote for your favorite!

## **Concept A**
![Screenshot from 2023-05-04 14-57-21](https://user-images.githubusercontent.com/1538838/236303312-05f38422-6de7-4d02-8dac-6cca1b19c05e.png)

## **Concept B**
![Screenshot from 2023-05-04 14-57-46](https://user-images.githubusercontent.com/1538838/236303393-f4e7971d-ef2c-457b-8d26-9452ea7d90f6.png)

## **Concept C**
![Screenshot from 2023-05-04 14-58-00](https://user-images.githubusercontent.com/1538838/236303436-4bf384cd-1662-4749-b40e-7c35f0d1a982.png)

## **Concept D**
![Screenshot from 2023-05-04 14-58-17](https://user-images.githubusercontent.com/1538838/236303487-0f64dac2-2a88-4cef-aad4-66d1b31b30eb.png)

## **Concept E**
![Screenshot from 2023-05-04 14-58-28](https://user-images.githubusercontent.com/1538838/236303520-e3fe6a74-9cbb-4ef2-b585-84286d847352.png)

**Comment by adambkaplan** [comment]
Worth noting - these designs are a "v1", and are not final. E is the current leading logo, followed by A and B.

Personally - E was my least favorite. The "checkered flag" to me doesn't have a strong relationship to the mission of Results. B was my favorite because it connotes filing/archiving, followed by A and D (tied). I could get behind a logo that blends B with A and/or D.

@wlynch @bobcatfish @afrittoli can vouch for what these logos looked like in large format at cdCon.

**Comment by sayan-biswas** [comment]
For me also concept E didn't make much sense. Since the project mostly represent data being archived, the background symbols can be more like a database or file or combination.

**Comment by manuelwallrapp** [comment]
I like concept E and C. They are both simple and catchy. E because it is the final state of a pipelinerun.

**Comment by adambkaplan** [comment]
Here is v2 from the CDF:

![image](https://github.com/tektoncd/results/assets/1538838/6ffcc805-7427-427d-bbe1-d85e76c212c9)

@manuelwallrapp unfortunately I recommended a merger of concepts A and B before your comment. To your point - the current iteration feels cluttered and lacks the simplicity of E and C. The other ["additional artwork" icons](https://github.com/cdfoundation/artwork/tree/main/tekton/additional-artwork) likewise have less clutter.

My thoughts:

- The "folder" can be made smaller so only the top is visible.
- The two extra lines near the cat's neck can be removed.
- Padding around the cat's face doesn't appear to be consistent (especially around the ears).
- Remaining icons can be adjusted to achieve a balanced look and feel.

**Comment by adambkaplan** [comment]
Version 3 was just published:

![image](https://github.com/tektoncd/results/assets/1538838/3cb9734e-a7ef-4bc6-9710-7476750fffd8)

Speaking for myself - I really like it! Very happy with this.

**Comment by adambkaplan** [comment]
Cleaning up this old discussion - version 3 was adopted! :tada: 

Official logo is here: https://github.com/cdfoundation/artwork/tree/main/tekton/additional-artwork/tekton-results
